protocol/communities_events_token_master_test.go

```diff
@@ -219,6 +219,8 @@ func (s *TokenMasterCommunityEventsSuite) TestReceiveRequestsToJoinWithRevealedA
 }
 
 func (s *TokenMasterCommunityEventsSuite) TestTokenMasterAcceptsRequestToJoinAfterMemberLeave() {
+	s.T().Skip("flaky test")
```

Comment:
Tests are re-run in the CI, so flaky tests shouldn't be blocking (i.e., they shouldn't cause the test job to turn red). Did this test fail on every re-run?